sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeHealthcareEntitiesOptions.java

```diff
@@ -43,7 +43,8 @@ public AnalyzeHealthcareEntitiesOptions setIncludeStatistics(boolean includeStat
     /**
      * Set the value of {@code disableServiceLogs}.
      *
-     * @param disableServiceLogs The default value of this property is 'false'. This means, Text Analytics service logs
+     * @param disableServiceLogs The default value of this property is 'false', except for methods like
```

Comment:
Can you change the message according to the class you are in? 
Example, instead of having a generic message like this, maybe in the `AnalyzeHealthcareEntitiesOptions` class say that the default is `true`.

I know in .NET we can't do it easily, so not sure Java

Reply:
For setter, It is not. but for getter, it is possible in TA